### DISCUSSION TITLE
nocodb-sdk [Dec 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4666] [Discord Threads]
> 🙋‍♂️ **lublak**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### nocodb-sdk has API SDK & some common types used in both backend and frontend.  
#  
> 🙋‍♂️ **lublak**  
#### Ah! Thanks for the answer.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1039920577244303440)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
